### PR TITLE
use v6 of node

### PR DESCRIPTION
**Pull request by bjfletcher**
up to the latest and greatest release of Node

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -102,6 +102,6 @@
     ]
   },
   "engines": {
-    "node": "4.4.2"
+    "node": ">=6.4.0"
```
Comment (ironsidevsquincy) [inline package.json]: It's annoying that you can use semver style in circle.yml, but I'd set this as exactly `6.4.0` (just so we know tests and app are running exactly the same versions)

**Comment by andygnewman** [comment]
Can you let us know when you merge it in, I'd suggest that we then give it a couple of hours before merging anything else in, in case there are any performance / memory issues that arise out of putting it into prod.

File: package.json (hunk 1 above)
Comment (bjfletcher) [inline package.json]: Done.

By the way... if all our Next apps use different minor versions of Node in v4 or v6 then we probably should have some kind of automatic Node version switching when we `cd` into a project, don't you think?

Comment (ironsidevsquincy) [inline package.json]: I uses https://github.com/Kikobeats/nodengine

**Pull request merged**